src/constants.js

```diff
@@ -24,4 +24,17 @@ module.exports = {
     'TV Station',
     'Train Station',
   ],
+  REBIRTHLOCATIONS: [
```

Comment:
```suggestion
  REBIRTH_LOCATIONS: [
```

Lets also change the `LOCATIONS` above to `VERDANSK_LOCATIONS`